scala_proto/scala_proto.bzl

```diff
@@ -18,579 +18,61 @@ load(
-    )
-
-    native.bind(
-        name = "io_bazel_rules_scala/dependency/proto/netty_transport",
-        actual = "@scala_proto_rules_netty_transport//jar",
-    )
-
-    native.maven_jar(
-        name = "scala_proto_rules_netty_resolver",
-        artifact = "io.netty:netty-resolver:4.1.32.Final",
-        sha1 = "3e0114715cb125a12db8d982b2208e552a91256d",
-        server = "scala_proto_deps_maven_server",
-    )
-
-    native.bind(
-        name = "io_bazel_rules_scala/dependency/proto/netty_resolver",
-        actual = "@scala_proto_rules_netty_resolver//jar",
-    )
-
-    native.maven_jar(
-        name = "scala_proto_rules_netty_common",
-        artifact = "io.netty:netty-common:4.1.32.Final",
-        sha1 = "e95de4f762606f492328e180c8ad5438565a5e3b",
-        server = "scala_proto_deps_maven_server",
-    )
-
-    native.bind(
-        name = "io_bazel_rules_scala/dependency/proto/netty_common",
-        actual = "@scala_proto_rules_netty_common//jar",
-    )
-
-    native.maven_jar(
-        name = "scala_proto_rules_netty_handler_proxy",
-        artifact = "io.netty:netty-handler-proxy:4.1.32.Final",
-        sha1 = "58b621246262127b97a871b88c09374c8c324cb7",
-        server = "scala_proto_deps_maven_server",
-    )
-
-    native.bind(
-        name = "io_bazel_rules_scala/dependency/proto/netty_handler_proxy",
-        actual = "@scala_proto_rules_netty_handler_proxy//jar",
-    )
-
-    native.maven_jar(
-        name = "scala_proto_rules_opencensus_api",
-        artifact = "io.opencensus:opencensus-api:0.18.0",
-        sha1 = "b89a8f8dfd1e1e0d68d83c82a855624814b19a6e",
-        server = "scala_proto_deps_maven_server",
-    )
-
-    native.bind(
-        name = "io_bazel_rules_scala/dependency/proto/opencensus_api",
-        actual = "@scala_proto_rules_opencensus_api//jar",
-    )
-
-    native.maven_jar(
-        name = "scala_proto_rules_opencensus_contrib_grpc_metrics",
-        artifact = "io.opencensus:opencensus-contrib-grpc-metrics:0.18.0",
-        sha1 = "8e90fab2930b6a0e67dab48911b9c936470d43dd",
-    )
-
-    native.bind(
-        name = "io_bazel_rules_scala/dependency/proto/opencensus_contrib_grpc_metrics",
-        actual = "@scala_proto_rules_opencensus_contrib_grpc_metrics//jar",
-    )
-
-def _root_path(f):
-    if f.is_source:
-        return f.owner.workspace_root
-    return "/".join([f.root.path, f.owner.workspace_root])
-
-def _colon_paths(data):
-    return ":".join([
-        "{root},{path}".format(root = _root_path(f), path = f.path)
-        for f in sorted(data)
-    ])
-
-def _retained_protos(inputs, blacklisted_proto_targets):
-    blacklisted_protos = [dep for target in blacklisted_proto_targets for dep in target.proto.transitive_sources]
-    blacklisted_protos_dict = dict(zip(blacklisted_protos, blacklisted_protos))
-    return [f for f in inputs if blacklisted_protos_dict.get(f, None) == None]
-
-def _valid_proto_paths(transitive_proto_path):
```

Comment:
not needed anymore?

Reply:
nope, this is about simplifying the handling of the externals